src/coreclr/jit/codegen.h

```diff
@@ -261,13 +261,13 @@ class CodeGen final : public CodeGenInterface
     // Prolog functions and data (there are a few exceptions for more generally used things)
     //
 
-    void genEstablishFramePointer(int delta, bool reportUnwindData);
-#if defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
-    void genFnPrologCalleeRegArgs();
-#else
-    void genFnPrologCalleeRegArgs(regNumber xtraReg, bool* pXtraRegClobbered, RegState* regState);
-#endif
-    void genEnregisterIncomingStackArgs();
+    void      genEstablishFramePointer(int delta, bool reportUnwindData);
+    void      genHomeRegisterParams(regNumber initReg, bool* initRegStillZeroed);
```

Comment:
The bool could be a return value, but seems ok this way too.

Reply:
Also most functions are using this pattern so it seems good to be consistent.